bee-block/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-beta.8 - 2022-XX-XX
+
+### Changed
+
+- Updated dependencies;
+
```

Comment:
Should have more

Reply:
Yes, doing my own review and writing it at the same time 👍🏻